### ISSUE TITLE
Build instructions

### ISSUE THREAD

**Issue by xdxita** [xuhongv/PHY6252_6222_SDK#1]
Can you please provide build instruction for this SDK?
Where can we find the compiler?
Is there any windows environment for this?

Thanks!

**Comment by Sugiie** [comment]
@xdxita Did You Get any information On this?
Actually I am New To This Module, If You Can Help Me With Your Experience.

Thanks In Advance

**Comment by xdxita** [comment]
> @xdxita Did You Get any information On this? Actually I am New To This Module, If You Can Help Me With Your Experience.
> 
> Thanks In Advance

No, and I'm not using this module due to lack of information.

**Comment by Sugiie** [comment]
Yes I agree With You, Infact Manufacturer Don't Have Much Information On this.
Reason To use this is its Cheap Compared to other.
 
by the way Which Module You are using Now? @xdxita
